kafkazk/brokers.go

```diff
@@ -154,6 +160,11 @@ func (b BrokerList) SortByID() {
 	sort.Sort(brokersByID(b))
 }
 
+// SortByIDDesc sorts the BrokerList by ID values.
+func (b BrokerList) SortByIDDesc() {
```

Comment:
Handy shortcut for the future - `sort.Reverse` is a stdlib primitive for this: `sort.Sort(sort.Reverse(brokersByID(brokers)))`

Reply:
✅ I'll replace with that